src/common/base/Base.h

```diff
@@ -67,7 +67,7 @@
 
 #include "common/base/Logging.h"
 
-#define MUST_USE_RESULT __attribute__((warn_unused_result))
+#define NG_MUST_USE_RESULT __attribute__((warn_unused_result))
```

Comment:
Why change this ?

Reply:
Some extra notes:

Our macro MUST_USE_RESULT conflicts with s2's third-party's macro, so I change it to NG_MUST_USE_RESULT